db/migrate/20230222155433_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorId
+      t.string :title
+      t.text :text
+      t.datetime :updatedAt
+      t.datetime :createdAt
```

Comment:
- [ ] Kindly delete these columns, these are not required because 👉🏽 `t.timestamps` will crate updated_at and created_at columns